.github/workflows/docs.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         run: |
           cd docs
           python3 -mpip install -r requirements.txt
+          # force installing Jinja2<3.1.0 to avoid issues with Sphinx
+          # File "/usr/local/lib/python3.8/dist-packages/sphinx/util/rst.py", line 21, in <module>                                                                                              
+          #   from jinja2 import Environment, environmentfilter
+          # ImportError: cannot import name 'environmentfilter' from 'jinja2'
+          pip install --upgrade 'Jinja2<3.1.0'
```

Comment:
Can we just add this to our requirements.txt?

Reply:
And leave a comment for it linking to https://github.com/sphinx-doc/sphinx/issues/10306